### PR TITLE
UI login screen fix

### PR DESCRIPTION
**Pull request by ctanveer**
…component

### Description

Login screen ui fixed. Made minor changes to button and text-input component 

### Link to ticket

*Please add a link to the GitHub issue being addressed by this change.*

### Type

- [x] Fix
- [ ] Feature
- [ ] Cleanup 

### Screenshot of the result

*If your change affects the user interface, you should include a screenshot of the result with the pull request.*

### CODE REVIEW

File: app/Domain/Auth/Templates/login.blade.php [hunk 1]
```diff
@@ -56,7 +58,7 @@
         <?php $tpl->dispatchTplEvent('beforeSubmitButton'); ?>
 
         <div class="">
-            <x-global::forms.button type="submit" content-role="primary" name="login">
+            <x-global::forms.button type="submit" content-role="primary" name="login" cornerShape="full" :takeFullWidth="true">
                 {{ $tpl->language->__('buttons.login') }}
```
Comment (marcelfolaron) [inline app/Domain/Auth/Templates/login.blade.php:62]: for the full width attribute can we just use "scale" and add full-width as an option

File: app/Domain/Auth/Templates/login.blade.php [hunk 2]
```diff
@@ -56,7 +58,7 @@
         <?php $tpl->dispatchTplEvent('beforeSubmitButton'); ?>
 
         <div class="">
-            <x-global::forms.button type="submit" content-role="primary" name="login">
+            <x-global::forms.button type="submit" content-role="primary" name="login" cornerShape="full" :takeFullWidth="true">
```
Comment (marcelfolaron) [inline app/Domain/Auth/Templates/login.blade.php:61]: I assume the cornershape about the rounded corners? Let's take this out and keep the buttons consistent.

File: app/Domain/Auth/Templates/login.blade.php [hunk 3]
```diff
@@ -72,10 +74,11 @@
     <?php if ($tpl->get('oidcEnabled')) { ?>
         <?php $tpl->dispatchTplEvent('beforeOidcButton'); ?>
         <div class="">
-            <center class="uppercase"><?php echo $tpl->language->__("label.or"); ?></center><br />
-            <a href="{{ BASE_URL }}/oidc/login" style="width:100%;" class="btn btn-primary">
-            <?php echo $tpl->language->__("buttons.oidclogin"); ?>
-            </a>
+            <br /><center class="uppercase"><?php echo $tpl->language->__("label.or"); ?></center><br />
+            
+            <x-global::forms.button onclick="window.location.href = '{{ BASE_URL }}/oidc/login'" content-role="primary" name="oidclogin" cornerShape="full" :takeFullWidth="true">
```
Comment (marcelfolaron) [inline app/Domain/Auth/Templates/login.blade.php:79]: same here, lets take out corner-shape and make takeFullWidth a value for scale

**Review by marcelfolaron — CHANGES_REQUESTED** [review]
Overall good but we want to avoid adding special rules for styling like border radius. The buttons should all be consistent in their design. In this case we are going to keep the new button design.

File: app/Domain/Auth/Templates/login.blade.php (hunk 2 above)
Comment (marcelfolaron) [reply app/Domain/Auth/Templates/login.blade.php:61]: class="w-full"

File: app/Domain/Auth/Templates/login.blade.php (hunk 1 above)
Comment (ctanveer) [reply app/Domain/Auth/Templates/login.blade.php:62]: removed cornerShape and takeFullWidth attribute. using class="w-full"

File: app/Domain/Auth/Templates/login.blade.php (hunk 2 above)
Comment (ctanveer) [reply app/Domain/Auth/Templates/login.blade.php]: done

File: app/Domain/Auth/Templates/login.blade.php (hunk 3 above)
Comment (ctanveer) [reply app/Domain/Auth/Templates/login.blade.php]: done

File: app/Views/Templates/components/forms/text-input.blade.php [hunk 4]
```diff
@@ -38,6 +37,10 @@
         $width = 'max-w-xs';
         $sizeClass = $size ? 'input-'.$size : '';
 
+    } elseif ($variant === 'fullWidth') {
+        $style= 'input-bordered  input-sm';
+        $width = 'w-full';
+        $sizeClass = $scale ? 'input-'.$scale : '';
     } else {
```
Comment (ctanveer) [inline app/Views/Templates/components/forms/text-input.blade.php:44]: for text-input component, I added a new variant called "fullWidth" as using the default did not take the full width of the input as it had both "w-full" and "max-w-xs". Would this be ok?

**Review by marcelfolaron — APPROVED** [review]

**Pull request merged**